sphinxserve/__init__.py

```diff
@@ -6,21 +6,25 @@
 
 __version__ = '0.7.4'
 __author__ = 'Daniel Mizyrycki'
```

Comment:
If it's helpful, I can positively confirm that no gevent-related bugs were caused by re-ordering the imports to match PEP-8 recommendations; it might be more accurate to say "can, but didn't".